helm-charts/mqserver/RELEASENOTES.md

```diff
@@ -8,6 +8,11 @@
 
 # Version History
 
+## Unreleased
```

Comment:
We should bump the version to 1.10.0

Reply:
Bump version also in  [helm-charts/mqserver/Chart.yaml at line 32](https://github.com/IBM/spm-kubernetes/pull/71/files#diff-ad8e928f3b55ab4a7764b79885c775600a691dbf884754a553361cd661087a3dR32)